### ISSUE TITLE
not rails 3.1 compatible

### ISSUE THREAD

**Issue by gucki** [hmcfletch/encrypted-cookie-store#2]
The gemspec forces rails 3.0.x dependencies so it doesn't work with rails 3.1. I'm not sure not if only the gemspec needs to be updated or if the rails apis changed too much.

**Comment by hmcfletch** [comment]
Figured since I hadn't tested it with 3.1, that I should ship it with inferred compatibility.  I'll check this out soon.  I haven't looked at 3.1 yet, so I'll have to do some more testing.

**Comment by hmcfletch** [comment]
I did some poking around in the 3.1 session code, and there seems to be a lot of changes in that area.  I'm going to keep the dependency 0.1 and 0.2 to be rails 3.0.\* for now.  I'll open a new milestone and assign this for 0.3.  Not sure when I'll get around to this though...

**Comment by hmcfletch** [comment]
I looked into this again and the code works just fine with Rails 3.1, but get the dependencies to work across 3.0, the release candidates and Rails 3.1 with the gemspec file has been a bit trickier.  It's the way ruby Gem::Specification handles the non-numeric point releases (i.e., 3.1.0.rc4) that is preventing me from easily adding the 3.1 dependencies.

**Comment by gucki** [comment]
Why not simply change to the following, as most gems do? :)

```
s.add_dependency('rails', '>= 3.0.0')
```

**Comment by hmcfletch** [comment]
Maybe I wasn't testing it right.  I am sure that will work just fine once 3.1 makes it out of Release Candidates, but for some reason it wasn't working with the 3.1.0.rc4 version I had installed.  It kept saying that it couldn't find it's dependencies.  I'll go ahead and push out 0.3 with the above dependency or a '~> 3.0'.  I'll check everything again once I get back from Florida.  Turns out traveling drastically cuts down on one's ability to release updates... Sorry if it's hindering your development...
